README.md

```diff
@@ -1,34 +1,58 @@
-## Welcome
+# Welcome
 
-### About this project
-This project enables you to create Cloudinary URLs for your images.
-Using this SDK, you can apply advanced transformations to your images.
+## About this project
 
+This project enables you to create Cloudinary URLs for your images and videos.
+Using this SDK, you can apply advanced transformations to your images and videos.
 
-### What is this project for?
-This project comes to replace and modernize our existing JS offering, and it's part of a larger effort in Cloudinary. 
+## Help & Examples
+- {@tutorial gettingStarted} 
+- {@tutorial annotatedExamples}
+- {@tutorial configuration}
 
-You can read more about the project here:
-- <a href="https://cloudinary.com/blog/cloudinary_s_next_generation_developers_sdks">Cloudinary's Next Generation SDKs</a>
-- <a href="https://cloudinary.com/blog/get_ready_for_cloudinary_s_next_generation_javascript_sdks">Cloudinary's next generation JS SDKs</a>
+## Installation
+```bash
+npm install @cloudinary/base 
+```
 
+## Simple usage
+```javascript
 
+// Import the cloudinary class
+import {Cloudinary} from '@cloudinary/base';
 
+// Create your instance
+const cld = new Cloudinary({
+    cloud: {
+        cloudName: 'demo'
+    },
+    url: {
+        secure: true // force https, set to false to force http
+    }
+});
 
-### Key Features
-- ESM packages, import only what you need
-- Declarative Syntax 
-- Fully typed for excellent IDE support
-- Easy to write, easy to read.
+// Lets create a new image
+const myImage = cld.image('sample');
+import {Resize} from '@cloudinary/base/actions/resize';
+myImage.resize(Resize.scale().width(100).height(100));
+
+// When we're done, we can apply all our changes and create a URL.
+const myURL = myImage.toURL();
+console.log(myURL);
+// https://res.cloudinary.com/demo/image/upload/c_scale,w_100,h_100/sample
```

Comment:
```suggestion
// https://res.cloudinary.com/demo/image/upload/c_scale,h_100,w_100/sample
```

Reply:
The returned URL would have the transformations in alphabetical order, I believe.